project/settings/development.py

```
@@ -0,0 +1,16 @@
+from project.settings.base import *
```

Ruff (0.8.2)

`from project.settings.base import *` used; unable to detect undefined names

(F403)